src/elements/content-sidebar/activity-feed/comment/__tests__/RepliesToggle.test.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, screen } from '@testing-library/react';
+import RepliesToggle from '../RepliesToggle';
+
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
+}));
+
+const showReplies = jest.fn();
+const hideReplies = jest.fn();
+const shownCount = 1;
+const totalCount = 9;
+
+const getWrapper = props =>
+    render(
+        <IntlProvider locale="en">
+            <RepliesToggle
+                id="123"
+                isRepliesLoading={false}
+                onShowReplies={showReplies}
+                onHideReplies={hideReplies}
+                repliesShownCount={shownCount}
+                repliesTotalCount={totalCount}
+                {...props}
+            />
+        </IntlProvider>,
+    );
+
+describe('elements/content-sidebar/ActivityFeed/comment/RepliesToggle', () => {
+    test('should correctly render show toggle', () => {
```

Comment:
Do we also need tests for when `isAlwaysExpanded` and/or `isRepliesLoading` is true?